Plugins/Gerrit/FormGerritPublish.cs

```diff
@@ -78,6 +78,19 @@ private bool PublishChange(IWin32Window owner)
             if (!string.IsNullOrEmpty(topic))
                 targetBranch += "/" + topic;
 
+            string reviewers = _NO_TRANSLATE_Reviewers.Text.Trim();
+            if (!string.IsNullOrEmpty(reviewers))
+            {
+                List<string> reviewersParam = new List<string>();
+                foreach (string reviewer in reviewers.Split(' '))
+                {
+                    if (!string.IsNullOrEmpty(reviewer))
```

Comment:
You can do a number of tweaks here:
* `if` can be merged into the `forearch` construct
```
foreach (string reviewer in reviewers.Split(' ').Where(r => !string.IsNullOrEmpty(r)))
````
* the whole block can be re-written with Linq 
```
var reviewersParam = reviewers.Split(' ')
                              .Where(r => !string.IsNullOrEmpty(r))
                              .Select(r => "r=" + r);
```

Reply:
Will apply this proposition. Fixed.